modules/conversion-center/classes/render/core-render.php

```diff
@@ -134,10 +134,25 @@ private function render_icons(): void {
 					'rel'        => 'noopener noreferrer',
 					'target'     => '_blank',
 				] );
+
 				?>
 				<div <?php echo $this->widget->get_render_attribute_string( 'icon-' . $key ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
 					<a <?php echo $this->widget->get_render_attribute_string( 'icon-link-' . $key ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-						<?php Icons_Manager::render_icon( $icon['icon_icon'], [ 'aria-hidden' => 'true' ] ); ?>
+						<?php
+						$mapping = Link_In_Bio::get_mapping( $icon['icon_platform'] );
+						$icon_lib = explode( ' ', $mapping )[0];
+						$library = 'fa-solid';
+						if ( 'fab' === $icon_lib ) {
+							$library = 'fa-brands';
+						}
```

Comment:
```suggestion
						$library = 'fab' === $icon_lib ? 'fa-brands' : 'fa-solid';
```
Maybe?

Reply:
@louiswol94 sure, I thought I saw in your onboarding doc that it was better to use explicit ifs rather then ternary, but will change it.